meteor/client/lib/EditAttribute.tsx

```diff
@@ -913,6 +919,121 @@ const EditAttributeJson = wrapEditAttribute(
 		}
 	}
 )
+const EditAttributeArray = wrapEditAttribute(
+	class EditAttributeArray extends EditAttributeBase {
+		constructor(props) {
+			super(props)
+
+			this.handleChange = this.handleChange.bind(this)
+			this.handleBlur = this.handleBlur.bind(this)
+			this.handleEscape = this.handleEscape.bind(this)
+		}
+		isArray(strOrg: string): { parsed: any[] } | false {
+			if (!(strOrg + '').trim().length) return { parsed: [] }
+
+			const values: any[] = []
+			const strs = (strOrg + '').split(',')
+
+			for (const str of strs) {
+				// Check that the values in the array are of the right type:
+
+				if (this.props.arrayType === 'boolean') {
+					const parsed = JSON.parse(str)
+					if (typeof parsed !== 'boolean') return false // type check failed
+					values.push(parsed)
+				} else if (this.props.arrayType === 'int') {
+					const parsed = parseInt(str, 10)
+
+					if (Number.isNaN(parsed)) return false // type check failed
+					values.push(parsed)
+				} else if (this.props.arrayType === 'float') {
+					const parsed = parseFloat(str)
+					if (Number.isNaN(parsed)) return false // type check failed
+					values.push(parsed)
+				} else {
+					// if (this.props.arrayType === 'string')
+					let parsed: string
+					try {
+						parsed = JSON.parse(str)
```

Comment:
What's the point of the `JSON.parse` here? Wouldn't it be enough to just cast `str` to string? Should probably be quicker than `JSON.parse`, especially if it actually is a long string and it will be trying to decide if it's an object or not.

Reply:
Good point! I think I was aiming for something that handled the `"asdf","qwerty"` case, but it really doesn't made sense after you pointed it out. Casting to string is the way to go 👍